pages/linux/efibootbgr.md

```diff
@@ -0,0 +1,25 @@
+# efibootmgr
+
+> Manipulate the UEFI Boot Manager (the Bootoptions).
+>
```

Comment:
Looks like this blank line is throwing off the linter. Perhaps we can remove it?